scss/_base_reset.scss

```diff
@@ -241,6 +241,11 @@
     max-width: 100%; /* 1 */
     padding: 0; /* 3 */
     white-space: normal; /* 1 */
+    
+    &.is-required::before {
```

Comment:
The top of the file says `// DO NOT edit this file except to upgrade Normalize in it's entirety` :)

This is a CSS that comes form external reset library. It's meant to provide a clean state. No class names or specific styling should be defined here.

Please move the legend styling to `_base_form.scss`.